app/src/main/java/uk/co/neuralcubes/neuralates/ControlFragment.java

```diff
@@ -191,4 +230,29 @@ public String apply(Robot bender) {
             }
         });
     }
+
+    private void enableSpheroActions(){
```

Comment:
Idea: instead of having separate methods to enable or disable a fixed set of views, you can have a generic method that takes a Collection<View> and an 'enabled' flag and applies it to all the elements of the collection.